src/main/java/me/imoltres/bbu/data/team/BBUTeam.kt

```diff
@@ -33,6 +37,17 @@ class BBUTeam(val colour: BBUTeamColour) {
 
     val players: MutableSet<BBUPlayer> = HashSet()
 
+    var bukkitTeam: Team? = null
```

Comment:
If all this is for is to change the formatting of their name in chat, then it is better to use a listener and update the format of messages.

Reply:
If I am not mistaken Teams are used for nametags. Not all to certain if Player#playerListName or Player#displayName is satisfactory, addition testing is needed to know.

Reply:
Nametags can be handled in the same way as the old bbu plugin, we just used a library, unsure how it worked but i'll look into adding it

Reply:
How it was, was fine using teams is the universal standard for them. So the way I had it would be fine.

Reply:
Yea, but it's a lot more effort than just using a library 👍

Reply:
Not really lol